ManatiCR/retopais
# RP-8: Blog node

https://manati.atlassian.net/browse/RP-8

How-to-test:
- Create 1 blog item with an embedded image
- Create (or generate) some other blog items
- Go to see the first created blog item
- It should look like the prototype (in a single column): https://www.estudiomanati.com/prototipos/retopais/#p=articulo&c=1
